### ISSUE TITLE
How to import to Postybirb

### ISSUE THREAD

**Issue by Nomaxice** [SpottedSqueak/FA-Gallery-Downloader#20]
I'm sorry if it's supposed to be obvious, but, while I was used the postybirb export button on the downloader, I'm not too sure how to import stuff into postybirb properly, I tried figuring out how to import the files and their descriptions/names etc. to posybirb+ but the folder import does not take the json files with the image files, and the template import needs templates already created prior in postybirb+, rather than import from a json, as far as I'm able to find...
Am I missing something?
It might be helpful having a short bit of info on how to use that feature!

**Comment by SpottedSqueak** [comment]
Oh no, did Postybirb change something? I'll try to take a look this weekend... If I remember right, you need to import each numbered folder and not the overall folder. Though what version of Postybirb are you using?

**Comment by Nomaxice** [comment]
I went to import each numbered folder, selecting the gallery folder for each, while the image was imported, no title, description etc. is imported, which would be the actually important part to publish back to another site, I'm not sure if it's supposed to grab the json infos or if it's intended to be such and I gotta then manually set back titles and everything?
Using postybirb+ 3.1.47 curently, though, regardless, updated recently

**Comment by SpottedSqueak** [comment]
No that's not the intention... I'll take a look at what Postybirb might've changed. This was working several versions ago!

**Comment by Nomaxice** [comment]
Gotcha, it did seem weird to have the json with everything, and have postybirb just ignore those entirely, heck it created separate entries for story PDF files and their matching thumbnails like they're different uploads so, I was rather confused there xD

Thank you for your time, hopefully it's an easy enough fix!

**Comment by Nomaxice** [comment]
Wait, update, I had a bit of a ponder based on what you said, I first had tried to import the "exports" folder which, didn't work, so I assumed that I needed to import the folders where the files are, which means I went to [exports, my name, number, gallery] and imported that. Since you mentioned importing the numbered folder I tried and, that worked, I didn't expect that the behavior was a sort of oddball specific of, can't take too high up, but also can't just take the "gallery" folder.

This is a behavior that may be very much worth documenting on the main page as without infos on it, it was a lil' confusing how it would behave knowing the usual computer behavior of either "go check all subfolders" or "only check the current folder files but not the subfolders", not this kinda in between!

**Comment by SpottedSqueak** [comment]
Oh, yeah, there's actually a limit to how many files you can import to Postybirb at once, so when I export the files I break them up into 50 submission chunks since that works better! Hence the numbered folders. You're right though, I should really make sure to document that somewhere.

**Comment by Nomaxice** [comment]
Yeah, specifically am surprised that, importing the numbered folder works, but importing the gallery folder within the numbered folder does not! This is what caught me off guard there

**Comment by SpottedSqueak** [comment]
That's more due to Postybirb's functionality, I've had to work within the constrains of that... I'll see about adding in some messaging for that though.

**Comment by Nomaxice** [comment]
Fair! But yeah a note on the github description about how to use the PB export/import and perhaps on the program's UI (like a little help button to show the "how to") would pretty much fix the somewhat unexpected workings!

Still, other than that, worked great once I figured that was the issue, and PB is happily posting away the FA stuff to sofurry to catch up to my other galleries, that said, I noticed there's a little "originally posted X", perhaps have a toggle during export on that? I'm gonna manually remove those, especially as I already keep track of art dates based on when the piece was gotten, don't much care for when it was posted to FA originally!
